src/languages/en.ts

```diff
@@ -1021,8 +1021,8 @@ export default {
     },
     privateNotes: {
         title: 'Private notes',
-        personalNoteMessage: 'Keep notes about this chat here. You are the only person who can add, edit or view these notes.',
-        sharedNoteMessage: 'Keep notes about this chat here. Expensify employees and other users on the team.expensify.com domain can view these notes.',
+        personalNoteMessage: 'Keep notes about this chat here. You're the only person who can add, edit, or view these notes.',
```

Comment:
Hey @jamesdeanexpensify. When a sentence includes a single quote `'`, we have to use a double for the sentence container. Can you change the sentence so that it starts and ends with `"` just like [this sentence](https://github.com/Expensify/App/blob/133502a6aff8ac15e62c91fcc47fdde32f2083af/src/languages/en.ts#L424) does? Thanks